tap_quickbooks/schemas/time_activities.json

```diff
@@ -1,9 +1,10 @@
 {
   "properties": {
     "HourlyRate": {
+      "format": "singer.decimal",
       "type": [
         "null",
-        "integer"
+        "string"
```

Comment:
Same for HourlyRate, if it is quantity then it won't be in decimal.

Reply:
Reverted back changes for HourlyRate.